libvmaf/src/feature/integer_adm.c

```diff
@@ -0,0 +1,211 @@
+/**
+ *
+ *  Copyright 2016-2020 Netflix, Inc.
+ *
+ *     Licensed under the BSD+Patent License (the "License");
+ *     you may not use this file except in compliance with the License.
+ *     You may obtain a copy of the License at
+ *
+ *         https://opensource.org/licenses/BSDplusPatent
+ *
+ *     Unless required by applicable law or agreed to in writing, software
+ *     distributed under the License is distributed on an "AS IS" BASIS,
+ *     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *     See the License for the specific language governing permissions and
+ *     limitations under the License.
+ *
+ */
+
+#include <errno.h>
+#include <math.h>
+#include <string.h>
+
+#include "feature_collector.h"
+#include "feature_extractor.h"
+
+#include "integer_adm_function.h"
+#include "mem.h"
+#include "adm_options.h"
+
+#ifndef NUM_BUFS_ADM
+#define NUM_BUFS_ADM 30
+#endif
+
+#define PROFILE_INTEGER_ADM 1
+
+#if PROFILE_INTEGER_ADM
```

Comment:
Same here

Reply:
Thanks for review, we will update the code and push a new commit